client/state/ui/test/reducer.js

```diff
@@ -29,7 +29,8 @@ describe( 'reducer', () => {
 			'olark',
 			'preview',
 			'actionLog',
-			'happychat'
+			'happychat',
+			'mediaModal'
```

Comment:
I used to be against them but am now for them. Python has allowed them as part of the language which is nice. Clean diffs are nice, and consistency is nice for me. I still remove the trailing comma if it's on a single line (function call, etc…).